src/puppetlabs/trapperkeeper/services_internal.clj

```diff
@@ -58,6 +58,13 @@
    :dependencies (schema/pred sequential?)
    :fns-map FnsMap})
 
+(def ProtocolandDependenciesMap
+  "Schema defining the map used to represent the protocol and dependencies
+  of a service."
+  {:fns (schema/pred seq?)
+   :dependencies (schema/pred vector?)
```

Comment:
This could also probably be made to be more strict, since we know exactly what syntaxes are legal binding forms for the dependencies.  Binding forms can look like this:

```clj
[[:SomeService some-fn other-fn]
 OtherService]
```

So you could do something like `[BindingForm]`, and then define the `BindingForm` schema as something that used schema/cond or schema/or to represent the two legal forms, which would probably be just `schema/Symbol` for the second case, and then something along the lines of `[(schema/one schema/Keyword) schema/Symbol]` for the first case.